bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -90,31 +87,6 @@ public ItemChannelLink update(ItemChannelLink element) {
         return linkedItems;
     }
 
-    /**
-     * Returns a set of bound things for the given item name.
-     *
-     * @deprecated Will be removed soon. Use {@link ItemThingLinkRegistry#getLinkedThings(String)} instead.
-     *
-     * @param itemName
-     *            item name
-     * @return set of bound things for the given item name
-     */
-    @Deprecated
```

Comment:
You are right, the comment is not correct, but isn't that exactly what we talked about in the referenced issue (this was my initial question)?


The function is deprecated and now removed because there shouldn't be a bound thing for an item anymore (see Kai's response in the referenced issue).